Dockerfile

```diff
@@ -1,11 +1,10 @@
 # ---- Base Image ----
-ARG base=hmctspublic.azurecr.io/base/node:14-alpine
-
-FROM ${base} as base
+FROM hmctspublic.azurecr.io/base/node:14-alpine as base
 USER root
```

Comment:
You can remove "USER root" from here as the default user is root

Reply:
Removing the user root does not work, gets error after removing.

Reply:
oh ok